src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
 
+export interface Figure {
+  shape: Shape,
+  color: 'red' | 'green' | 'blue',
```

Comment:
create separate type for Color
```suggestion
  color: Color,
```